content/help_centre/github.md

```diff
@@ -0,0 +1,17 @@
+This section contains useful links for those using the library but who are unfamiliar using or navigating GitHub.  
```

Comment:
Can we remove the links and incorporate them in text that describes what they explain, I would write the instructions as:

Using Github
This section contains useful links for anyone unfamiliar with Github, it provides some pointers to documentation that will help you use and navigate the system.

For general guidance consult the [Github documentation](https://docs.github.com/en) in the first instance 

Releases

Consult the [Github guidance on navigating releases](https://docs.github.com/en/repositories/releasing-projects-on-github/searching-a-repositorys-releases) to get an understanding of identifying where the release information can be found and how to search for a specific release.

View the [Github release comparison documentation](https://docs.github.com/en/repositories/releasing-projects-on-github/comparing-releases) to get information on comparing different release versions and understanding how to use Github to determine the changes between releases.

Reply:
on it :)